libraries/codegen-tooling/inc/can_msg_defs.h

```diff
@@ -54,5 +54,7 @@ typedef enum {
   SYSTEM_CAN_MESSAGE_SOLAR_DATA_REAR = 46,
   SYSTEM_CAN_MESSAGE_LINEAR_ACCELERATION = 51,
   SYSTEM_CAN_MESSAGE_ANGULAR_ROTATION = 52,
-  NUM_SYSTEM_CAN_MESSAGES = 31
+  SYSTEM_CAN_MESSAGE_LIGHTS_SIGNAL_TYPES = 53,
```

Comment:
CAN messages are priority-based. ping @ckitagawa for comments on modifications to codegen-tooling

Reply:
Yeah, I'll talk to him.